packages/react-celo/src/screens/wallet-connect.tsx

```diff
@@ -88,7 +88,7 @@ export const WalletConnect = ({ onSubmit, provider }: Props) => {
 
   if (!provider) return null;
   let content = null;
-  if (!uri || loading) {
+  if (!uri || loading || error) {
```

Comment:
need to be able to show errors that are returned from wallet after URI exists. such as "chain not support"